include/grpcpp/impl/codegen/interceptor_common.h

```diff
@@ -219,10 +219,28 @@ class InterceptorBatchMethodsImpl
   // Alternatively, RunInterceptors(std::function<void(void)> f) can be used.
   void SetCallOpSetInterface(CallOpSetInterface* ops) { ops_ = ops; }
 
-  // Returns true if no interceptors are run. This should be used only by
-  // subclasses of CallOpSetInterface. SetCall and SetCallOpSetInterface should
-  // have been called before this. After all the interceptors are done running,
-  // either ContinueFillOpsAfterInterception or
+  // Returns true if the interceptors list is empty
```

Comment:
I think this is only valid after `SetCall` has been called, just like in your other comments about `RunInterceptors`, correct?

Reply:
This code and `RunInterceptors` are so similar that it screams of a need for code reuse.

Reply:
I'm sure some template magic could do the job, but we can leave it alone for now.